src/main/java/ch/myniva/gradle/caching/s3/internal/AwsS3BuildCacheServiceFactory.java

```diff
@@ -16,20 +16,20 @@
 
 package ch.myniva.gradle.caching.s3.internal;
 
-import static com.amazonaws.util.StringUtils.isNullOrEmpty;
```

Comment:
Please do not change order of imports.

Reply:
Sorry, now I disabled auto organize imports for this project.